dvc/remote/local/slow_link_detection.py

```diff
@@ -31,13 +31,12 @@ def __call__(self, *args, **kwargs):
             and self.should_display()
         ):
             msg = (
-                "You can cut execution time considerably. Check:\n"
+                "You can cut execution time considerably by using a different"
+                " `cache.type` configuration.\nSee "
                 "{blue}https://dvc.org/doc/commands-reference/config#cache{"
-                "reset}"
-                "\nfor "
-                "more information.\nTo disable this message, run:\n'dvc "
-                "config "
-                "cache.slow_link_warning False'".format(
+                "reset} for more information.\n"
+                "To disable this message, run:\n"
+                "'dvc config cache.slow_link_warning false'".format(
```

Comment:
I also updated the slow link warning message in accordance with the updates in https://github.com/iterative/dvc.org/pull/331

Reply:
Woops, I had forgotten to update the `TestSlowLinkDetectorDecorator` test so the message matches so this build wasn't passing but fixed it now @shcheklein ! Please see if it can be merged.

p.s. I also rebased as suggested by @efiop  in https://github.com/iterative/dvc/pull/2003#issuecomment-492385913 so the history in the PR is now shown below and cleaner. I gotta say it was kind of painful: had to resolve conflicts on multiple rebase steps (whereas with merge you only resolve once) but that may be because I had previously merged in my recent history, so I'll keep giving rebase a chance.